charts/kubewarden-controller/Chart.yaml

```diff
@@ -23,9 +23,9 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.0.3-rc3
+version: 2.1.0-rc1
```

Comment:
Ouch, did we forget to bump the version of the chart from `2.0.3-rc3` to `2.0.3`?

Leaving that aside, I think the new number proposed by the bot is wrong. We should use `2.0.4-rc1`